ShopAppTests/App/Account/Controllers/AddressFormViewControllerSpec.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  AddressFormViewControllerSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 2/23/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import RxSwift
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class AddressFormViewControllerSpec: QuickSpec {
+    override func spec() {
```

Comment:
You also need to test:
- binding of text fields/pickers with view model's variables
- pickers' item select actions (if it possible)
- pickers' custom data and text that setup with data from view model's subscription